Dockerfile

```diff
@@ -65,7 +65,7 @@ RUN --mount=type=bind,from=wheels,source=/wheels,target=/wheels \
     gnupg \
     wget \
     procps \
-    unzip tzdata libxml2 xz-utils \
+    unzip locales tzdata libxml2 xz-utils \
```

Comment:
Just so you know, locales would get installed anyway as a dependency of the hwaccel packages. Installing before ensures it's installed from the regular repos and not from the testing repos.